hack/lib/golang.sh

```diff
@@ -476,6 +476,7 @@ EOF
   local go_version
   IFS=" " read -ra go_version <<< "$(GOFLAGS='' go version)"
   local minimum_go_version
+  # TODO(cpanato): Need to switch this to 1.20 once we update images to newer go version
```

Comment:
also need to comment out the line in dependencies.yaml that is expecting this to be 1.20rc1

> `ERRO /home/prow/go/src/k8s.io/kubernetes/build/dependencies.yaml indicates that golang: 1.<major> should be at version 1.20rc1, but the following files didn't match: hack/lib/golang.sh`